packages/openapi-to-graphql/src/oas_3_tools.ts

```diff
@@ -472,22 +484,23 @@ export function getSchemaTargetGraphQLType(
 
 /**
  * Determines an approximate name for the resource at the given path.
+ *
+ * Remove the path parameters from the path.
+ *
+ * For example, turn '/user/{userId}/car' into 'userCar'.
+ *
+ * Note that the returned string is in camelCase.
  */
 export function inferResourceNameFromPath(path: string): string {
-  /**
-   * Remove the path parameters from the path
-   *
-   * For example, turn /user/{userId}/car into userCar
-   */
-  let pathNoPathParams = path.split('/').reduce((path, part) => {
+  const pathNoPathParams = path.split('/').reduce((path, part) => {
     if (!/{|}/g.test(part)) {
       return path + capitalize(part)
     } else {
       return path
     }
   })
 
-  return pathNoPathParams
+  return uncapitalize(pathNoPathParams)
```

Comment:
I was getting some strange behavior from this which caused problem in other places in the code. I am enforcing the lowercase letter for now but I'm not sure if this is the right way about going at it.

Reply:
Remove this.